Vic2ToHoI4/Source/Mappers/Graphics/GraphicsCultureGroup.h

```diff
@@ -24,6 +24,7 @@ class GraphicsCultureGroup
 	[[nodiscard]] const auto& getMaleMonarchPortraits() const { return maleMonarchPortraits; }
 	[[nodiscard]] const auto& getFemaleMonarchPortraits() const { return femaleMonarchPortraits; }
 	[[nodiscard]] const auto& getLeaderPortraits() const { return leaderPortraits; }
+	[[nodiscard]] const auto& getFemalePortraits() const { return femalePortraits; }
```

Comment:
Add tests for this. See MapperTests/Graphics/GraphicsCultureGroupTests.cpp for examples.